tests/framework/PostProcessors/SampleSelector/basic.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" ?>
+<Simulation verbosity="debug">
+  <RunInfo>
+    <WorkingDir>Basic</WorkingDir>
+    <Sequence>Sample,Process</Sequence>
+    <batchSize>1</batchSize>
+  </RunInfo>
+
+  <TestInfo>
+    <name>framework/PostProcessors/SampleSelector/Basic</name>
+    <author>gm</author>
+    <created>2018-09-12</created>
+    <classesTested>PostProcessors.SampleSelector</classesTested>
+    <description>
+      This test checks basic functionalities of the Sample Selector
```

Comment:
can you add a better description of what this test does? E.g. The Model XXX generate samples based on this equation (XXX), we get the minimum.

In addition, since this PP get the maximum and minimum you should add 2 steps...one for the minimum and one for the maximum (I know that internally the methods called are the same, but the user can take this test as baseline to develop his on input deck)